PokemonAPI/PokemonAPI/NLog.config

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      autoReload="true"
+      throwExceptions="false"
+      internalLogLevel="Off" internalLogFile="C:\Users\prome\Desktop\NetCoreWebAPICourse\PokemonAPI\PokemonAPI\Logs\nlog-internal.log">
```

Comment:
Don't use **absolute** file paths. That is generally bad practice because from environment to environment things are different

Say you have C:\Users\Matija Zagar\NetCoreWebAPICourse\Log.txt

If I run this app it will create this path on my computer..
I don't need this and don't want this to happen

Better practice is to use **relative** paths
Where **mostly** ./ defines current working directory... and you use ../ to go up one level

`..\..\log\internal-nlogtxt` 
so this would end up going 2 levels above where the code is executing and create log folder there and txt file inside.

This will make it so that it's the same on each computer

Say your code is in NetCoreWebAPI/API/nlog.Config if you go one level up to NetCoreWebAPI and put logs there everyone who clones and runs app will have logs in root of the application NetCoreWebAPI/Logs..

Bottom line... test and try it. Define log location with **relative** path instead of **absolute**